WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerViewModel.kt

```diff
@@ -263,8 +263,15 @@ class MediaPickerViewModel @Inject constructor(
         } else if (domainModel?.isLoading == true) {
             PhotoListUiModel.Loading
         } else {
+            val stringId = when (if (domainModel?.mediaTypes?.size === 1) domainModel.mediaTypes.first() else null) {
```

Comment:
I'm with you on this one. Android studio kept bugging me to remove the extracted val. Thought it might cause linting issues if I ignored the suggestion. I'll change this.